spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /posts' do
+    context 'Get #index' do
+      before(:each) do
+        get '/users/7/posts'
```

Comment:
- [ ] All your tests are failing due to a routing error. There is no route defined in your current routes file. Kindly fix this error and ensure all works as expected.

  ![image](https://github.com/sam2if/BLOGAPP/assets/36542986/b4658f04-b7f9-4792-99b5-e0b08d313392)

  ![image](https://github.com/sam2if/BLOGAPP/assets/36542986/2fbd5168-24f9-4cf5-af04-775ebb1e547c)

Reply:
- Well resolved
<img width="529" alt="Screenshot 2024-01-26 at 16 05 57" src="https://github.com/sam2if/BLOGAPP/assets/58692590/72cc795e-9cf5-435b-a803-c7ea0533271c">